controllers/start.go

```diff
@@ -1,116 +1,60 @@
 package controllers
 
 import (
-	"fmt"
 	"os"
-	"os/exec"
-	"os/signal"
-	"syscall"
 
-	"github.com/opencontainers/runc/libcontainer/system"
 	"go.uber.org/zap"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/kubernetes"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
 	ctrl "sigs.k8s.io/controller-runtime"
 	ctrlzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
 
 	forgev1alpha1 "github.com/dominodatalab/forge/api/v1alpha1"
-	"github.com/dominodatalab/forge/internal/builder"
-	"github.com/dominodatalab/forge/internal/message"
-	_ "github.com/dominodatalab/forge/internal/unshare"
-	"github.com/dominodatalab/forge/plugins/preparer"
 	// +kubebuilder:scaffold:imports
 )
 
-const leaderElectionID = "forge-leader-election"
-
 var (
 	newScheme = runtime.NewScheme()
 	setupLog  = ctrl.Log.WithName("setup")
 )
 
-func StartManager(namespace string, metricsAddr string, enableLeaderElection bool, brokerOpts *message.Options, preparerPluginsPath string, enableLayerCaching bool, debug bool) {
-	reexec()
+const leaderElectionID = "forge-leader-election"
 
+func StartManager(cfg Config) {
 	atom := zap.NewAtomicLevel()
-	if debug {
+	if cfg.Debug {
 		atom.SetLevel(zap.DebugLevel)
 	}
 
 	logger := ctrlzap.New(func(opts *ctrlzap.Options) {
 		opts.Level = &atom
+		opts.Development = true
```

Comment:
`opts.Development = cfg.Debug`?

Reply:
I guess you said this is a TODO anyway... ignore me.

Reply:
yup, but still a good thing to point out.